charts/astronomer/templates/houston/houston-bootstrap-role.yaml

```diff
@@ -1,7 +1,7 @@
 ################################
 ## Houston Bootstrap Role
 #################################
-{{- if and .Values.global.rbacEnabled (and (not .Values.houston.backendSecretName) (not .Values.houston.backendConnection)) }}
+{{- if and .Values.global.rbacEnabled (and (not .Values.houston.airflowBackendSecretName) (not .Values.houston.airflowBackendConnection)) }}
```

Comment:
Is this change to repair an existing bug in our chart, or it related to this PR? Do we need only need this role for airflow metadata db secrets?

Reply:
I think this is actually a fix. We never hit it because we don't set either, but yes, even if we don't merge the rest of this PR, we should PR this line right @andriisoldatenko ?

Reply:
Well actually these 3 files.